joomla/joomla-cms
# Fix the calendar to work in the subform field

### Summary of Changes
This patch fix new calendar field to work in the subform field

### Testing Instructions
Make sure that calendar field work in the subform 
Example xml, can place in "Custom HTML" module xml (somwhere in params fieldset):
```xml
<field name="items" type="subform"  multiple="true"  label="Subform Field" >
  <form>
    <field name="something"  type="calendar" label="Calendar" />
  </form>
</field>
```

### Documentation Changes Required
none
